chromadb/test/property/test_cross_version_persist.py

```diff
@@ -345,6 +348,18 @@ def test_cycle_versions(
         embedding_function=not_implemented_ef(),  # type: ignore
     )
 
+    # Should be able to clean log immediately after updating
+    embeddings_queue = system.instance(SqliteDB)
+
+    # Cleaning the log is dependent on vector segments migrating their max_seq_id from the pickled metadata file to SQLite.
+    # Vector segments migrate this field automatically on init, but at this point the segment has not been loaded yet.
+    trigger_vector_segments_max_seq_id_migration(
+        embeddings_queue, system.instance(SegmentManager)
+    )
+
+    embeddings_queue.clean_log(coll.id)
```

Comment:
exercises the max_seq_id migration--if the migration doesn't happen or fails, `.clean_log()` will have no effect (because the max_seq_id for one of the segments is missing) and the invariant check immediately below will fail if the # of embeddings added is > sync threshold